blockwork/config/blockwork.py

```diff
@@ -18,13 +18,16 @@
 @dataclass
 class Blockwork:
     project      : str
+    site         : str                 = "./site.yml"
```

Comment:
In the docs everything so far has been `.yaml` not `.yml` - we should be consistent, I don't mind which way